recipes/mqc/run_test.pl

```diff
@@ -0,0 +1,11 @@
+# Test file for testing Perl modules installation
+$|=1;
```

Comment:
Interesting, what happens if you try this? I never had a package with perl and python at once I guess. However, if this does not work you can also use `python -c "import numpy"`